CorsixTH/Lua/hospitals/player_hospital.lua

```diff
@@ -761,6 +761,11 @@ function PlayerHospital:makeVipEndFax(vip_rating, name, cash_reward, vip_message
   self.world.ui.bottom_panel:queueMessage("report", message, nil, Date.hoursPerDay() * 20, 1)
 end
 
+--! Give the player information with the UIInformation dialog
+function PlayerHospital:giveInfo(message)
+  self.world.ui:addWindow(UIInformation(self.world.ui, { message }))
```

Comment:
I don't know what other Hospital code is doing, but `UIInformation` is a UI thing rather than a hospital thing, so this leaks UI details to hospital code.

Likely it happens all over the place, and is it not something you can easily fix. As such, it's probably better to keep this for now,

`world.ui` has similar problems, but that is even more wide-spread. No danger to run out of code to refactor thus :)

Reply:
Anything and everything seems to call ui in terms of needing to generate a message of some kind. Did this come from a misunderstanding of what I said about the adviser message previously? I was fine to give the cheats dialog ability to create messages, they just shouldn't use `_A` for something that always uses `_S`